crates/hyperswitch_domain_models/src/payments/payment_intent.rs

```diff
@@ -440,6 +446,247 @@ impl From<PaymentIntentUpdate> for PaymentIntentUpdateInternal {
     }
 }
 
+use diesel_models::PaymentIntentUpdate as DieselPaymentIntentUpdate;
+
+impl From<PaymentIntentUpdate> for DieselPaymentIntentUpdate {
```

Comment:
Can we remove this implementation (and `diesel_models::PaymentIntentUpdate`), and have the necessary implementations on the domain model itself?

Reply:
Yes.

Reply:
This will be a significant change involving the kv layer. This can be taken up in a separate issue